packages/lodestar-cli/package.json

```diff
@@ -65,6 +65,7 @@
     "deepmerge": "^3.2.0",
     "ethers": "^5.0.2",
     "expand-tilde": "^2.0.2",
+    "ganache-core": "^2.10.2",
```

Comment:
dev dependency? 
Maybe we should use something more typesafe and with less problematic dependenvies like builderevm?

Reply:
+1 for buidler

Reply:
Sorry, meant to clean up some of that ganache-core code that i was using in the setup and enforce type safety.  Does it look better to you now?

Reply:
My interpretation of your comment @mpetrunic was that you were commenting on the way I used the dependency (i.e. not as a dev dep) and used `any` previously for the server listen/close methods.  If that's an incorrect assumption on my part, then what is problematic and/or not typesafe about ganache-core?

Reply:
builderevm is alternative to ganache-core so I'm suggesting to swap them since ganache-core doesn't have types or they don't work and has some native dependencies that are hard compile on some OS

Reply:
ok cool.  just swapped ganache-core for buidler in dc0d18843c9d411d976550254ed152fa65960774